lib/valueflow.cpp

```diff
@@ -106,16 +106,16 @@ static void bailoutInternal(TokenList *tokenlist, ErrorLogger *errorLogger, cons
         fileName = fileName.substr(p + 1);
     }
     ErrorLogger::ErrorMessage errmsg(callstack, tokenlist->getSourceFilePath(), Severity::debug,
-        std::string(fileName) + ":" + std::to_string(static_cast<unsigned long long>(line)) + ":" + std::string(function) + " bailout: " + what, "valueFlowBailout", false);
+        std::string(fileName) + ":" + MathLib::toString(static_cast<unsigned long long>(line)) + ":" + std::string(function) + " bailout: " + what, "valueFlowBailout", false);
```

Comment:
it is redundant to copy fileName (and function) to a temporary std::string

Reply:
Thanks, that was an oversight from me.